javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -15,7 +15,9 @@ const BasicCourseTable = ({ classes, checks, displayQuarter, allowExport }) => {
   const COLOR_UNAVAILABLE = availabilityColors.UNAVAILABLE;
   const COLOR_CLOSEFULL = availabilityColors.CLOSEFULL;
   const COLOR_AVAILABLELECTUREORCLASSWITHSECTIONS = {backgroundColor: '#CEDEFA'};
-  const COLOR_AVAILABLESECTION = {backgroundColor: '#EDF3FE'};
+  const COLOR_AVAILABLESECTION = {backgroundColor: '#EDF3FE', fontStyle: 'italic'};
+  const COLOR_UNAVAILABLESECTION = {backgroundColor: '#FF8080', fontStyle: 'italic' };
+  const COLOR_CLOSEFULLSECTION = {backgroundColor: '#FFD761', fontStyle: 'italic' };
```

Comment:
I wonder whether it's a good idea to put code that changes the font into a variable called `COLOR_AVAILABLESECTION`.

(Hint: No, it isn't. :-) )

What would be better style is to 
1. add a meaningful CSS class such as `className="courseSection"` to the elements that are sections.
2. In the file App.css, add code that defines all sections as having `fontStyle: 'italic'`

This should not be merged in this form.

Reply:
We will create a new issue to address these changes.